<?xml version="1.0" encoding="utf-8"?>
<launch>
    <arg name="gz_gui" default="false"/>
    <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="world_name" value="$(find multi_turtlebot3_simulation)/worlds/corrider.world"/>
        <arg name="paused" value="false"/>
        <arg name="use_sim_time" value="true"/>
        <arg name="headless" value="false"/>
        <arg name="gui" value="$(arg gz_gui)"/>
        <arg name="debug" value="false"/>
    </include>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find multi_turtlebot3_simulation)/launch/multi_robot.rviz" required="true"/>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_0"/>
        <arg name="robot_x_pos" value="0.0"/>
        <arg name="robot_y_pos" value="0.0"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_1"/>
        <arg name="robot_x_pos" value="-0.8"/>
        <arg name="robot_y_pos" value="-1.6"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_2"/>
        <arg name="robot_x_pos" value="-0.8"/>
        <arg name="robot_y_pos" value="-0.8"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_3"/>
        <arg name="robot_x_pos" value="-0.8"/>
        <arg name="robot_y_pos" value="0.0"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_4"/>
        <arg name="robot_x_pos" value="-0.8"/>
        <arg name="robot_y_pos" value="0.8"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_5"/>
        <arg name="robot_x_pos" value="-0.8"/>
        <arg name="robot_y_pos" value="1.6"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_6"/>
        <arg name="robot_x_pos" value="0.0"/>
        <arg name="robot_y_pos" value="-2.4"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_7"/>
        <arg name="robot_x_pos" value="0.0"/>
        <arg name="robot_y_pos" value="-1.6"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_8"/>
        <arg name="robot_x_pos" value="0.0"/>
        <arg name="robot_y_pos" value="-0.8"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_9"/>
        <arg name="robot_x_pos" value="-0.8"/>
        <arg name="robot_y_pos" value="-2.4"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_10"/>
        <arg name="robot_x_pos" value="0.0"/>
        <arg name="robot_y_pos" value="0.8"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
    <include file="$(find multi_turtlebot3_simulation)/launch/one_robot.xml">
        <arg name="tb3_name" value="tb_11"/>
        <arg name="robot_x_pos" value="0.0"/>
        <arg name="robot_y_pos" value="1.6"/>
        <arg name="robot_z_pos" value="0.0"/>
        <arg name="robot_yaw" value="0.0"/>
    </include>
</launch>
